ext/hash/hash_sha3.c

```diff
@@ -49,14 +49,14 @@ static inline uint64_t load64(const unsigned char* x) {
 	return ret;
 }
 static inline void store64(unsigned char* x, uint64_t val) {
-	char i;
+	unsigned char i;
 	for (i = 0; i < 8; ++i) {
 		x[i] = val & 0xFF;
 		val >>= 8;
 	}
 }
 static inline void xor64(unsigned char* x, uint64_t val) {
-	char i;
+	unsigned char i;
```

Comment:
I wonder if it isn't better to do ``size_t i`` instead of using a char here in both cases as it's meant to be an index.